.gitignore

```diff
@@ -2,7 +2,6 @@
 ## files generated by popular Visual Studio add-ons.
 
 .vs
-.vscode
```

Comment:
Can we leave this?

Reply:
@jterry75 I added that because unlike the `.vs`, it only contains the "build config" when you are using VSCode. It is just a couple of JSON files and are not machine specific. The `.vs` one has the extension index database which should be excluded.

Reply:
No if you have made any project specific settings/debug/etc it will be in .vscode. Its a pretty standard practice to .gitignore the .vscode folder

Reply:
Ok ok, I usually have it as a standard config to build/debug the project anywhere without machine or OS specific things. And all other project I've worked actually keep that on the repo so it is simples to build/debug. Think on that as a simple `build.sh`/ `run.sh`.

Bot ok, I understand your point. Will put that back when I get back to a PC and merge this.

Thanks!